contributors/design-proposals/apiserver-build-in-admission-plugins.md

```diff
@@ -0,0 +1,31 @@
+# Build some Admission Controllers into the Generic API server library
+
+**Related PR:**
+
+| Topic | Link |
+| ----- | ---- |
+| Admission Control | https://github.com/kubernetes/community/blob/master/contributors/design-proposals/admission_control.md |
+
+## Introduction
+
+At the moment admission controllers are implemented as plugins and they have to be compiled into the final binary in order to be used at a later time. To streamline this process especially for aggregated API servers we would like to build some plugins into the generic API server library.
+
+## Implementation
+
+The first step would be to move some plugins from [admission pkg](https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/admission) to the generic API server library. We can start off by moving the namespace lifcycle controller. 
```

Comment:
> i think i would prefer to ask why admins need to --admission-control=NamespaceLifecycle at all versus it just forever being on by default. now that there is no major disagreement anymore that namespaces are things that should impose a lifecycle process, we should think about removing any burden on operator to enable it.

I agree.  Right now its completely impossible to include it, so making it possible is a first step and this outlines a way to do that.